src/app/layout/Footer.tsx

```diff
@@ -1,5 +1,27 @@
-import { FooterContainer } from '@/app/layout';
+import { GridContainer, FooterContainer } from '@/app/layout';
+import * as S from '@/app/layout/Footer.style';
+import { LongGithubIcon, ProgrammersIcon } from '@/shared/assets';
+import logo from '@/shared/assets/image/logo.png';
 
 export const Footer = () => {
-  return <FooterContainer>Footer</FooterContainer>;
+  return (
+    <FooterContainer>
+      <GridContainer>
+        <S.LogoBox>
+          <img src={logo}></img>
```

Comment:
self-closing + alt 속성도 표현되면 좋겠네요!
```suggestion
          <img src={logo} />
```

Reply:
습관을 들여야겠네요! 감사합니다~ https://github.com/GrebbMe/grabbme-fe/commit/66774ecb616f2d7efdf6481d19f5fb049fa56620